features/fixtures/macos/macOSTestApp/MainWindowController.m

```diff
@@ -92,6 +92,7 @@ - (IBAction)useDashboardEndpoints:(id)sender {
 }
 
 - (IBAction)executeMazeRunnerCommand:(id)sender {
+    Scenario.baseMazeAddress = @"http://localhost:9339";
```

Comment:
So on mac we never load the address from disk?

Reply:
Yes, that's right.  Currently (and I think always) we only run the macOS e2e tests on the same machine that Maze Runner is running on, so the address is always localhost.